hack/generate-helm-release.sh

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+function echoerr {
+    >&2 echo "$@"
+}
+
+_usage="Usage: $0 --out <DIR>
+Package the Nephe chart into a chart archive.
+Environment variable VERSION must be set.
+        --out <DIR>                  Output directory for chart archive
+        --help, -h                   Print this message and exit
+
+You can set the HELM environment variable to the path of the helm binary you want us to
+use. Otherwise we will download the appropriate version of the helm binary and use it."
+
+function print_usage {
+    echoerr "$_usage"
+}
+
+function print_help {
+    echoerr "Try '$0 --help' for more information."
+}
+
+OUT=""
+
+while [[ $# -gt 0 ]]
+do
+key="$1"
+
+case $key in
+    --out)
+    OUT="$2"
+    shift 2
+    ;;
+    -h|--help)
+    print_usage
+    exit 0
+    ;;
+    *)    # unknown option
+    echoerr "Unknown option $1"
+    exit 1
+    ;;
+esac
+done
+
+if [ -z "$VERSION" ]; then
+    echoerr "Environment variable VERSION must be set"
```

Comment:
nit:
```suggestion
    echoerr "Environment variable $VERSION must be set"
```

Reply:
if we use $VERSION, that version will be substituted.. So we cannot use it